pinot-controller/src/main/java/com/linkedin/pinot/controller/validation/ValidationManager.java

```diff
@@ -264,120 +280,45 @@ public static long computeRealtimeTotalDocumentInSegments(
   }
 
   /**
-   * Computes a list of missing intervals, given a list of existing intervals and the expected frequency of the
-   * intervals.
+   * Computes missing intervals based on the given existing intervals and the expected frequency of the intervals.
```

Comment:
Can you briefly add the explanation on how we define missing intervals here?

Reply:
Added